test/features/formItemExtra.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import faker from 'faker';
+
+import { Tester } from '@mighty-justice/tester';
+
+import { COMPONENT_GENERATORS } from '../factories';
+import { IValue } from '../../src/props';
+
+const SUPPORTING_COMPONENTS = [
+  'Form',
+  'FormCard',
+  'FormDrawer',
+  'FormModal',
+];
+
+function extraRenderer (value: IValue) {
```

Comment:
Can shorten this rendered significantly with a ternary: `return value ? <span>...</span> : null;`